CHANGELOG.md

```diff
@@ -24,12 +24,25 @@ Initial release of sanger-tol/readmapping, created with the [nf-core](https://nf
 
 ### `Deprecated`
 
-## v1.1 - [date]
+## v1.1.0 – Hebridean Black - [2023-mm-dd]
```

Comment:
v1.1.0 should come first, as per https://keepachangelog.com/en/1.0.0/#how

Reply:
Updated